packages/@vivliostyle/theme-base/css/partial/footnote.css

```
@@ -0,0 +1,78 @@
+  --vs-footnote--area-before-display: block;
+  --vs-footnote--area-before-margin-block: var(--vs--hr-margin-block);
+  --vs-footnote--area-before-margin-inline: var(--vs--hr-margin-inline);
+  --vs-footnote--item-margin-block: 0;
+  --vs-footnote--item-margin-inline: 0;
+  --vs-footnote--root-counter-reset: vs-counter-footnote;
+}
+
+.footnote, [role=doc-footnote] {
+  counter-increment: vs-counter-footnote;
+
+  color: var(--vs-footnote--color-body, var(--vs-color-body));
+  font-family: var(--vs-footnote--font-family, var(--vs-font-family));
+  font-feature-settings: var(--vs-footnote--font-feature-settings, var(--vs-font-feature-settings));
+  font-kerning: var(--vs-footnote--font-kerning, var(--vs-font-kerning));
+  font-size: var(--vs-footnote--font-size, var(--vs-font-size));
+  font-stretch: var(--vs-footnote--font-stretch, var(--vs-footnote--font-stretch));
+  font-style: var(--vs-footnote--font-style, var(--vs-font-style));
+  font-synthesis: var(--vs-footnote--font-synthesize, var(--vs-font-synthesis));
+  font-variant: var(--vs-footnote--font-variant, var(--vs-font-variant));
+  font-variation-settings: var(--vs-footnote--font-variation-settings, var(--vs-font-variation-settings));
+  font-weight: var(--vs-footnote--font-weight, var(--vs-font-weight));
+  letter-spacing: var(--vs-footnote--letter-spacing, var(--vs-letter-spacing));
+  line-height: var(--vs-footnote--line-height, var(--vs-line-height));
+  text-align: var(--vs-footnote--text-align, var(--vs-text-align));
+  text-spacing: var(--vs-footnote--text-spacing, var(--vs-text-spacing));
+  word-spacing: var(--vs-footnote--word-spacing, var(--vs-word-spacing));
+}
+
+.footnote::footnote-call,
+[role=doc-footnote]::footnote-call {
+  content: var(--vs-footnote--call-content);
+  font-size: var(--vs-footnote--call-font-size);
+  margin-inline: var(--vs-footnote--call-margin-inline);
+  position: relative;
+  inset-block-start: var(--vs-footnote--call-inset-block-start);
+}
+
+.footnote::footnote-marker,
+[role=doc-footnote]::footnote-marker {
+  content: var(--vs-footnote--marker-content);
+  margin-inline: var(--vs-footnote--marker-margin-inline);
+}
+
+@media print {
+  .footnote, [role=doc-footnote] {
+    float: footnote;
+    margin-block: var(--vs-footnote--item-margin-block);
+    margin-inline: var(--vs-footnote--item-margin-inline);
+  }
+}
+
+@-adapt-footnote-area {
+  margin-block: var(--vs-footnote--area-margin-block);
+}
+
+/* prettier-ignore */
+@-adapt-footnote-area ::before {
+  display: var(--vs-footnote--area-before-display);
+  border-block-start-color: var(--vs-footnote--area-before-border-color, var(--vs--hr-border-color, var(--vs-border-color)));
+  border-block-start-width: var(--vs-footnote--area-before-border-width, var(--vs--hr-border-width, var(--vs-border-width)));
+  margin-block: var(--vs-footnote--area-before-margin-block);
+  margin-inline: var(--vs-footnote--area-before-margin-inline);
+} ;
```

```suggestion
}
```

ここにセミコロンがあるのは間違いでしょう。
Vivliostyle.jsではログに "E_CSS_SYNTAX ;" と警告が出力されます。